app/test/test_api_application.py

```diff
@@ -0,0 +1,43 @@
+from unittest.mock import patch, Mock
+
+import drucker_pb2
+from .base import BaseTestCase, create_app_obj, create_service_obj, create_eval_obj
+from io import BytesIO
+from models import db, EvaluationResult, Evaluation
+
+
+class ApiEvaluationTest(BaseTestCase):
+    """Tests for ApiEvaluation.
+    """
+
+    @patch('core.drucker_dashboard_client.drucker_pb2_grpc.DruckerDashboardStub')
+    def test_post(self, mock_stub_class):
+        mock_stub_obj = Mock()
+        mock_stub_obj.UploadEvaluationData.return_value = drucker_pb2.UploadEvaluationDataResponse(status=1, message='success')
+        mock_stub_class.return_value = mock_stub_obj
+        aobj = create_app_obj()
+        create_service_obj(aobj.application_id)
+        response = self.client.post(f'/api/applications/{aobj.application_id}/evaluation',
+                                    content_type='multipart/form-data',
+                                    data=dict(file=(BytesIO(b'my file contents'), "work_order.123")))
+        self.assertEqual(200, response.status_code)
+        self.assertEqual(response.json, {'status': True, 'evaluation_id': 1})
```

Comment:
> Not to upload the same evaluation data, separate evaluate and upload test data

At the moment, this PR does not include the duplication check for evaluation data. If we do it, we need to add that function to `reckurd` repository.

Anyway, we can add the duplication upload test here.

Reply:
@keigohtr 
I added duplication check for evaluation data in https://github.com/rekcurd/drucker-dashboard/pull/50/commits/78766ba60652bf6a79b349d739666941ace83c27